cmake/BuildParameters.cmake

```diff
@@ -124,14 +124,13 @@ if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "i386")
     #     - Only plugins. No package will link to them.
     set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
 
-    if (DISABLE_ADVANCE_SIMD)
-        set(ARCH_FLAG "-msse -msse2 -march=i686")
-    else()
+    if (NOT DISABLE_ADVANCE_SIMD)
```

Comment:
I don't see anything on the arch flag inside cmake/linux-compiler-i386-multilib.cmake
The typical usecases are 
1/ build on a machine that support avx for users with older CPU
2/ Disable AVX/SSE for valgrind or virtual box

SSE2 and March=i686 is the minimal acceptable optimization. It doesn't hurt gentoo as all peoples will use the march=native path anyway.

In my opinion, your build isn't fully portable.